basic/src/test/java/org/jboss/hal/testsuite/test/configuration/singleton/SingletonPolicyTestCase.java

```diff
@@ -139,11 +141,16 @@ public void editCacheContainer() throws Exception {
                 .navigateToTreeItem()
                 .clickLabel();
 
-        new ConfigChecker.Builder(client, SINGLETON_POLICY_ADDRESS)
-                .configFragment(page.getConfigFragment())
-                .editAndSave(ConfigChecker.InputType.TEXT, CACHE_CONTAINER, CACHE_CONTAINER_RESOURCE_EDIT)
-                .verifyFormSaved()
-                .verifyAttribute(CACHE_CONTAINER, CACHE_CONTAINER_RESOURCE_EDIT);
+        try {
+            new ConfigChecker.Builder(client, SINGLETON_POLICY_ADDRESS)
+                    .configFragment(page.getConfigFragment())
+                    .editAndSave(ConfigChecker.InputType.TEXT, CACHE_CONTAINER, CACHE_CONTAINER_RESOURCE_EDIT)
+                    .verifyFormSaved()
+                    .verifyAttribute(CACHE_CONTAINER, CACHE_CONTAINER_RESOURCE_EDIT);
+        } catch (NoSuchElementException e) {
+            //TODO remove this when issue is resolved
+            throw new UnexpectedException("This is failing because of https://issues.jboss.org/browse/HAL-1255", e);
```

Comment:
- IMO `java.rmi.UnexpectedException` is not appropriate. 
- I would suggest to instead of "This is failing because of ..." write something like "Possible cause of not found element might be ..." since we cannot be sure about cause of future error and should not automatically anticipate it has to be the same flaw.